virt/virt-2-5-release-notes.adoc

```diff
@@ -67,6 +67,10 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 * The Containerized Data Importer (CDI) can now import containerDisk storage volumes from the container image registry at a faster speed and allocate storage capacity more efficiently. CDI can pull a containerDisk image from the registry in about the same amount of time as it would take to import from an HTTP endpoint. You can import the disk into a PersistentVolumeClaim (PVC) equal in size to the disk image to use the underlying storage more efficiently.
 
 //CNV-7168 - DataVolume and storage diagnostics
+* It is now easier to diagnose and troubleshoot issues when preparing virtual machine (VM) disks that are managed by DataVolumes:
++
+** For asynchronous image upload, an error message is returned, before the connection is closed, if the size of the virtual disk image is larger than the size of the target DataVolume.
```

Comment:
I'm not a native speaker yet I think we could optimize this paragraph like 'For asynchronous image upload, if the size of the virtual disk image is larger than the size of the target DataVolume, an error message is returned before the connection is closed.' If the current description is universally acceptable from writing perspective, please ignore my dogmatic comment.

Reply:
Reworded for clarity